app/views/article/editor.js

```diff
@@ -100,10 +25,9 @@ define(['app', 'ck-editor', 'text!views/article/editor-directive.html', 'lodash'
                 if($route.current.$$route && $route.current.$$route.isNew)
                     $scope.document = {};
                 else{
-                    $q.when(genericService.get('v2017', 'articles', $route.current.params.id))
+                    return $q.when(genericService.get('v2017', 'articles', $route.current.params.id))
```

Comment:
this is very strange the have a service where you pass the version.

Reply:
the service was built with intentions of multiple ver's but i will change it to accept url instead of the params